object_tracker/object_tracker.py

```diff
@@ -99,5 +135,8 @@ def init_bounding_box(self, frame: ndarray, bounding_box: List[int]):
         logging.info("finish init tracker bounding box")
         logging.info(f"bounding box: {bounding_box}")
         self._init = True
-        if self._run_optical_flow:
+        if self._run_optical_flow or self.run_of_low_score:
             self._match.init_bounding_box(frame, bounding_box)
+
+    def get_parameters(self):
+        return self._tracker.get_parameters()
```

Comment:
At least some of the functionality here should be moved to a different branch and outside of the file for this version.